iceoryx_posh/test/moduletests/test_popo_condition_variable.cpp

```diff
@@ -87,6 +87,39 @@ TEST_F(ConditionVariable_test, NoNotifyResultsInNotBeingTriggered)
     EXPECT_FALSE(m_waiter.wasNotified());
 }
 
+TEST_F(ConditionVariable_test, NotifyOnceResultsInNoWaitSingleThreaded)
+{
+    m_signaler.notify();
+    m_waiter.wait();
+    // We expect that the next line is reached
+    EXPECT_TRUE(true);
```

Comment:
shouldn't the `WatchDog` be used here and also in the other tests with a blocking wait?

Reply:
You are right ... I already fixed this (slipped through when I merged with master).